h/h_api/exceptions.py

```diff
@@ -0,0 +1,61 @@
+from h.h_api.model.json_api import JSONAPIError, JSONAPIErrorBody
+
+
+class JSONAPIException(Exception):
+    """An exception which can turn itself into a a JSON API error response."""
+
+    http_status = None
+
+    def _error_bodies(self):
+        """Get the instances of JSONAPIErrorBody representing this error."""
+        raise NotImplementedError()
+
+    def as_dict(self):
+        """Get a JSON API compatible dict representing this error."""
+        return JSONAPIError.create(self._error_bodies()).raw
+
+
+class SchemaValidationError(JSONAPIException):
+    """Represent a number of schema validation errors.
+
+    This class can be used to build up the errors and then check if the
+    exception should be raised.
+    """
```

Comment:
Who is the intended audience for this docstring?

Reply:
Nobody, this is some old cruft, I'll chop it out.